### PR TITLE
v1.2 release

### PR DESCRIPTION
**Pull request by skriss**
Contains:
- the changelog
- the versioned docs site
- the blog post

cc @carlisia @prydonius @nrb @jonasrosland

### CODE REVIEW

File: site/_posts/2019-11-07-Velero-1.2-Sets-Sail.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+- @betta1
+- @lingtongj
+- @spiffcs
+- @s12chung
+- @boxcee
+- @andyzhangx
+- @sseago
+- @Frank51
+- @alexander
```
Comment (jonasrosland) [inline site/_posts/2019-11-07-Velero-1.2-Sets-Sail.md]: Can we add links to their GitHub pages?

File: site/_posts/2019-11-07-Velero-1.2-Sets-Sail.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+In version 1.2, Velero automatically detects when you are trying to clone an existing namespace, and clones the persistent volumes used by the namespace as well. This doesn’t require the user to specify any additional flags for the `velero restore create` command.  This change lets you fully achieve your goal of cloning namespaces using persistent storage within a cluster.
+
+## Improved Server-Side Encryption Support
+
+To help you secure your important backup data, we’ve added support for more forms of server-side encryption of backup data on both AWS and GCP. Specifically:
+- On AWS, Velero now supports Amazon S3-managed encryption keys (SSE-S3), which uses AES256 encryption, by specifying `serverSideEncryption: AES256` in a backup storage location’s config.
+- On GCP, Velero now supports using a specific Cloud KMS key for server-side encryption by specifying `kmsKeyName: <key name>` in a backup storage location’s config.
```
Comment (jonasrosland) [inline site/_posts/2019-11-07-Velero-1.2-Sets-Sail.md]: These don't show up as lists on the blog, add a line break before the list and it should work.

File: site/_posts/2019-11-07-Velero-1.2-Sets-Sail.md (hunk 1 above)
Comment (skriss) [reply site/_posts/2019-11-07-Velero-1.2-Sets-Sail.md]: done!

File: site/_posts/2019-11-07-Velero-1.2-Sets-Sail.md (hunk 2 above)
Comment (skriss) [reply site/_posts/2019-11-07-Velero-1.2-Sets-Sail.md]: thanks, fixed!

**Pull request merged**